Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs

```diff
@@ -70,6 +70,39 @@ public static class ChainDictionaries
             { Chain.TestnetBlastSepolia, "ETH" }
         };
 
+        public static Dictionary<Chain, string> NativeTokenAddressOf = new Dictionary<Chain, string>()
```

Comment:
Native tokens don't have addresses. These are all contract addresses for the wrapped version of the native token